examples/opentelemetry-example-app/setup.py

```diff
@@ -16,7 +16,7 @@
 
 setuptools.setup(
     name="opentelemetry-example-app",
-    version="0.1.dev0",
+    version="0.2a.0",
```

Comment:
```suggestion
    version="0.2a0",
```

We should use the normalized name. It is unusual to add a dots around the alphabetical component. See:

```python
In [1]: from packaging import version

In [2]: version.parse("0.2a.0")
Out[2]: <Version('0.2a0')>
```

In Dynatrace, we even added a check in the setup.py to ensure we only use normalized version strings (https://github.com/Dynatrace/OneAgent-SDK-for-Python/blob/a59b28cc9cdb7f818dee9304903d91df69eb31f0/setup.py#L71-L74)

Reply:
Good call, changed in d0e9f84.